posts/2025-02-17_theres_a_pharmaverse_package_for_that/managing-clinical-trial-data.qmd

```diff
@@ -0,0 +1,323 @@
+---
+title: "Managing Clinical Trial Data? There’s a Pharmaverse Package for That"
+author:
+  - name: Gift Kenneth
+  - name: Sunil Gupta
+  - name: APPSILON
+description: "Looking for R packages to manage clinical trial data? pharmaverse has tools for every stage from data collection to submission."
+date: "2025-02-17"
+# please do not use any non-default categories.
+# You can find the default categories in the repository README.md
+categories: [Technical, Community]
+# feel free to change the image
+image: "pharmaverse-post.png"
+---
+
+<!--------------- typical setup ----------------->
+
+```{r setup, include=FALSE}
+long_slug <- "2025-02-17_managing_clinical_trial_data..."
+# renv::use(lockfile = "renv.lock")
+```
+
+<!--------------- post begins here ----------------->
+
+Managing clinical trial data is no small task. It needs to be precise, compliant, and efficient. Traditionally, this meant using proprietary tools and working within siloed systems, which often made the process more complicated and expensive than necessary. But we think there’s a better way.
+
+The [**pharmaverse**](https://pharmaverse.org/) is an open-source ecosystem of R packages built specifically for clinical trials. These tools integrate seamlessly with the [Tidyverse](https://www.tidyverse.org/), making data management more flexible, efficient, and transparent.
+
+Whether you’re collecting, validating, analyzing, or preparing data for regulatory submission, there’s a pharmaverse package designed to support your workflow and help you work smarter.
+
+This post covers:
+
+-   Key stages of clinical trials and the R packages that support them
+
+-   Getting started with metadata-driven clinical reporting
+
+-   The process from raw data to ADaMs, including validation
+
+-   Creating ADSL datasets and essential programming steps
+
+-   Key players in Pharmaverse and whether you need all packages
+
+-   How pharmaverse compares to Tidyverse and how to learn it
+
+By the end, you'll have a clear understanding of how Pharmaverse supports clinical trial operations and how to apply these tools in your work.
+
+## **Key Stages of the Clinical Trial Process**
+
+Managing clinical trial data involves multiple stages, each with its own challenges. pharmaverse provides a range of packages for different parts of the process, sometimes even more than one option for the same task, so you can choose what works best for your needs rather than sticking to a single approach
+
+![](images/clinical-trial-process.png)
+
```

Comment:
Add a sentence (preferably in bold or in a highlighted box) clarifying here that you put together this list for the blog, and if any package was missed, we are happy to update the post to include it.  I wouldn't want anyone's package to feel left out.

Reply:
Or just say that these are some examples of pharmaverse packages and consult the site for the full list, as otherwise you're going to be maintaining this blog every time new packages get added